wp-content/themes/core/assets/css/src/theme/typography/lists.pcss

```diff
@@ -5,84 +5,108 @@
  * ----------------------------------------------------------------------------- */
 
 /* -------------------------------------------------------------------------
- * Global Lists
+ * Lists Shared
  * ------------------------------------------------------------------------- */
 
 ul,
 ol,
 dl {
 
-	.t-content & {
-		@mixin body-content;
-
-		margin-top: var(--spacer);
-		margin-bottom: var(--spacer);
+	.t-sink & {
+		@mixin t-body;
 	}
 }
 
-li {
+/* -------------------------------------------------------------------------
+ * List Item
+ * ------------------------------------------------------------------------- */
 
-	& ~ li {
-
-		.t-content & {
-			margin-top: var(--spacer-xs);
-		}
-	}
-}
+li {}
 
 /* -------------------------------------------------------------------------
- * UL
+ * Unordered List
  * ------------------------------------------------------------------------- */
 
-.list-bullet,
-.t-content ul {
+.t-list--bullets,
```

Comment:
have one extra dash, should be `.t-list-bullets`

Reply:
I see `bullets` as a modifier for the `list` block, hence the double dash. List could have other marker styles besides bullets, couldn't they?

Reply:
Hmm, true, I suppose these would be flavors of lists, which if that's the case, maybe it's weird that we use modifiers when they wouldn't necessarily build on one another and are instead straight up versions of a list (`.t-list-bullets` or `.t-list-styled`), it's not like we have a base `.t-list`, though perhaps we should?

Reply:
Yeah, I wonder about that, too. I'll go ahead and add a `t-list` selector to the shared styles block at the top of the file for consistancy.